docs/content/installation.md

```diff
@@ -266,24 +267,24 @@ This image is compressed with gzip to save space. In order to properly manage it
 
 ```bash
 backupobject add \
-  --uuid bo-lucid5d \
-  --display-name "Ubuntu 10.04 (Lucid Lynx) root partition" \
+  --uuid bo-ubuntu5d \
+  --display-name "Ubuntu 14.04 (Trusty Tahr) root partition" \
   --storage-id bkst-local \
-  --object-key ubuntu-lucid-kvm-md-32.raw.gz \
+  --object-key ubuntu-14.04.3-x86_64-30g-passwd-login-enabled.raw.tgz \
   --size 149084 \
   --allocation-size 359940 \
   --container-format gz \
```

Comment:
size, allocation-size and container-format need to be updated as well. The correct commands are in https://github.com/axsh/wakame-vdc/issues/688#issuecomment-147649610.